### PR TITLE
Cleanup for 2.18.0 release

### PR DESCRIPTION
**Pull request by jadahiya-MSFT**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Summarize the changes, including the goals and reasons for this change. Be sure to call out any technical or behavior changes that reviewers should be aware of.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1. Cleaning up changefiles and changelogs from the 2.18.0 release
2. Updating the hash values in our README

### CODE REVIEW

File: packages/teams-js/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+### Minor changes
+
+- Extended `RequestAppAudioHandlingParams` by adding `audioDeviceSelectionChangedCallback` for speaker selection updates
+
+### Patches
+
+- Added `meetingStage` and `settings` framecontexts to `clipboard`.
+- Added additional telemetry to `App`, `Dialog`, `GeoLocation`, `Location`, `Navigation`, `Pages`, and `Tasks` capabilities
+- Created new `MessageRequest` interface with required properties to enhance type-safety
+- Added telemetry to `barcode`, `calendar`, `call`, `clipboard`, `mail`, `marketplace` and `media` capabilities
```
Comment (AE-MS) [inline packages/teams-js/CHANGELOG.md:20]: Not a huge deal but perhaps the line two above this could be combined with this one?

**Review by AE-MS — APPROVED** [review]
:shipit:

File: packages/teams-js/CHANGELOG.md (hunk 1 above)
Comment (jadahiya-MSFT) [reply packages/teams-js/CHANGELOG.md:20]: Hmm we can make that change, but then the README would be out of sync with the one on the npm package

Comment (AE-MS) [reply packages/teams-js/CHANGELOG.md:20]: Then don't.  😶

**Review by erinha — APPROVED** [review]

**Pull request merged**